dist/blocks.js

```diff
@@ -11,6 +13,35 @@ export default ((editor, opts = {}) => {
   } // Add icon to mj-hero
 
 
+  //add button block for landing page 
```

Comment:
It looks to me like you added your changes to the dist/ folder? Please copy them to the file in the src/ folder. If you run `npm run build` the JS files will be built and copied to the dist/ folder.

Reply:
Let me fix this soon.

Reply:
I have removed the one from dist folder